### PR TITLE
Feature: Connect to redis database.

### PR DESCRIPTION
**Pull request by h-alice**
Now the backend can connect to redis database to storage visitor counter.

### CODE REVIEW

**Comment by h-alice** [comment]
Current site after successfully deployed.

image

**Comment by NatLee** [comment]
🥲🥲🥲

File: src/main_server.go [hunk 1]
```diff
@@ -1,25 +1,78 @@
...
+	// Load template.
+	page_template = template.Must(template.ParseFiles("/static/html/page_template.html"))
+	if page_template == nil {
+		log.Fatal("[x] Error loading template.")
+	}
+
 	http.HandleFunc("/", clientHandler)
 	log.Println("[+] Collecting God Boris Power...")
-	log.Println("[+] Server Starting Up.")
+	log.Println("[.] Server starting up and listening on port 8080.")
```
Comment (NatLee) [inline src/main_server.go:75]: Is the port 8080 can be set in `.env` file? :D

File: static/html/page_template.html [hunk 2]
```diff
@@ -34,5 +34,8 @@
             <span> ╚═════╝  ╚═════╝ ╚═════╝     ╚═════╝  ╚═════╝ ╚═╝  ╚═╝╚═╝╚══════╝</span>                                                                                                       
         </pre>
     </div>
+    <div class="counter">
+        <h2>You are the No.{{.Counter}} prayer to the God Boris.</h2>
```
Comment (NatLee) [inline static/html/page_template.html:38]: <(_ _)>

**Review by NatLee — COMMENTED** [review]
No any problems here.
It's good to go! :D

File: static/html/page_template.html (hunk 2 above)
Comment (h-alice) [reply static/html/page_template.html:38]: Sad doge branch...
image

**Pull request merged**